### DISCUSSION TITLE
[Feature] For "Album Updated" email notifications, allow a digest to be sent instead

### DISCUSSION THREAD

**Discussion by mblissmer** [immich-app/immich#13478] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

If this feature is enabled as it is currently, and you add a picture to a shared album, look through your photos, pick another, and keep doing that a few times, the person with notifications enabled receives an individual email for each photo you add.
They go check their email and have 15 different emails, sent over the course of five minutes.

It would be a nice option to have an end of day "multiple assets were added to this album" type email instead of a "per asset" email.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Related: #12424

**Comment by bo0tzz** [comment]
Fixed in #12223, will be in the next release
